manifest.json

```diff
@@ -3,7 +3,7 @@
   "short_name": "Better Subs",
   "manifest_version": 2,
   "name": "Better Subscriptions for YouTube™",
-  "version": "0.13.1.1",
+  "version": "0.14",
```

Comment:
pls change back, version is updated before release (though Ill probably change this process in the future)